doc/classes/ProjectSettings.xml

```diff
@@ -746,6 +746,9 @@
 		<member name="input_devices/pointing/ios/touch_delay" type="float" setter="" getter="" default="0.15">
 			Default delay for touch events. This only affects iOS devices.
 		</member>
+		<member name="internationalization/locale/include_text_server_data" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], text server break iteration rule sets, dictionaries and other optional data are included in exported project.
```

Comment:
```suggestion
			If [code]true[/code], text server break iteration rule sets, dictionaries and other optional data are included in the exported project.
```

Also, if we know it it might be worth mentioning what kind of size impact is to be expected when including it (though I guess it might be platform-specific).

Reply:
Also, are there some criteria we can document to help users understand when they should enable this? Like the major language families which **must** have this, and those which could benefit from it for advanced things (like I guess hyphenation even in latin languages).

Reply:
Added some notes about contents and size of data for built-in text servers.